elastalert/elastalert.py

```diff
@@ -776,7 +776,9 @@ def alert(self, matches, rule, alert_time=None):
         # Run the alerts
         alert_sent = False
         alert_exception = None
+        alert_pipeline = {}
         for alert in rule['alert']:
+            alert.pipeline = alert_pipeline
```

Comment:
Sharing the same object is the the point of this, so that the first alert can put information into it and the second can use that information. I'll add a comment.